lib/bokken_web/controllers/availability_controller.ex

```diff
@@ -16,10 +16,11 @@ defmodule BokkenWeb.AvailabilityController do
 
   def index(conn, availability_params) do
     availabilities = Events.list_availabilities(availability_params, [:mentor])
+    unavailabilities = Events.list_unavailabilities(availability_params, [:mentor])
 
     conn
     |> put_status(:ok)
-    |> render("index.json", availabilities: availabilities)
+    |> render("index.json", availabilities: availabilities, unavailabilities: unavailabilities)
```

Comment:
If you are not updating the view how does this work?

Reply:
I put all mentors related to an `event_id` in the same endpoint.

Reply:
You are passing `unavailabilities` to the view, but not properly rendering it.